src/iop/demosaic.c

```diff
@@ -733,8 +733,17 @@ void process(dt_iop_module_t *self,
   const gboolean tiling = _tiling_requirements(self, piece, roi_in, roi_out, dual, greens, direct, do_capture, FALSE, method,
                           &overlap, &valid_rows, &tile_height, &num_tiles);
 
+  const gboolean bad_tiling = tiling && (valid_rows < 30);
```

Comment:
Out of curiosity: where does this magic value of 30 come from?

Reply:
Thanks for the explanation.
I wonder whether we should document those magic values by using named constants or just plain comments.
This should make it easier to conserve your reasoning about the value you chose for other people who are going to work on this module in the future.